src/facade/dragonfly_listener.cc

```diff
@@ -267,6 +267,10 @@ void Listener::OnConnectionClose(util::Connection* conn) {
   }
 }
 
+void Listener::OnMaxConnectionsReached(util::FiberSocketBase* sock) {
```

Comment:
What calls this? It looks like nothing(?), so maybe move it to a different PR?

Reply:
helio framework calls this. it's a virtual hook